src/InstrumentationEngine.Lib/FileLoggerSink.cpp

```diff
@@ -183,7 +185,10 @@ void CFileLoggerSink::WritePrefix(_In_ LoggingFlags flags)
 #else
                 WCHAR szFormatted[MAX_PATH];
                 wcsftime(szFormatted, MAX_PATH, szFormat, &localTime);
-                fwprintf(pOutputFile, szFormatted);
+
+                // Static analysis will complain that the following line prints a non-constant
+                // string, but there are no inputs to the string. Ignore.
+                fwprintf(pOutputFile, szFormatted);  // lgtm[cpp/non-constant-format]
```

Comment:
Is this the same error that you fixed in https://github.com/microsoft/CLRInstrumentationEngine/pull/408/files#diff-57c5f94f29184e8ca0b109256bc0adbb292ef774a8ed031a882753cc2067ce54

Why fix it there but suppress it here?

Reply:
I was going to think about this, and potentially reconsider it.

For many of these, fixed them just in case they end up with format characters like '%' in the string. I didn't think that was possible here.  It's probably better to be safe than sorry.